### ISSUE TITLE
Should we implement SRV support for ActiveDirectory provider?

### ISSUE THREAD

**Issue by rnhurt** [StackExchange/dnscontrol#178]
Now that #136 has landed should SRV support be implemented for AD?  I know that SRV records are pretty important for AD but I don't exactly how they work.  It looks like they are dynamically generated/updated by the DomainControllers themselves so we might not want to explicitly create them.  

However, if that's the case, could we remove the warning about SRV records being ignored?  My AD domain has over 100 SRV records and seeing all that output really makes my heart jump.  :)

**Comment by captncraig** [comment]
Yeah, the ignores get real spammy. Maybe we should condense that to "xxx records ignored. see [some log] file for full iist"

**Comment by tlimoncelli** [comment]
Currently we only implement the record types that StackOverflow uses (yeah, that's lazy but, that's all we've needed). If you use other record types, please implement them. It would be great if all the integration tests worked for AD.  I get very nervous about making changes to that provider until they do.
